project/webapp/static/js/thread.js

```diff
@@ -680,9 +680,11 @@ cw.NewCiviView = BB.View.extend({
         // $('.responses').height($('#new-civi-box').height() + $('.responses-box').height());
 
         this.magicSuggestView = new cw.LinkSelectView({$el: this.$('#magicsuggest'), civis: this.options.parentView.civis});
+        this.magicSuggestBillView = new cw.LinkSelectBillView({$el: this.$('#magicsuggestbills'), civis: this.options.parentView.civis});
```

Comment:
Break this into multiple lines for readability.

Reply:
makes sense. sure.